api/BlobBuilder.json

```
@@ -6,27 +6,19 @@
         "support": {
           "chrome": {
             "version_added": "8",
-            "prefix": "Webkit"
+            "version_removed": "24",
+            "prefix": "WebKit"
           },
           "chrome_android": {
             "version_added": "18",
-            "prefix": "Webkit"
+            "version_removed": "24",
```

I think this needs to be bumped up to `"25"` to pass the tests

```suggestion
            "version_removed": "25",
```

---

(feel free to merge once this is fixed, if this is the only change)